addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -90,3 +153,81 @@ def _compute_display_name(self):
     def _compute_avatar_128(self):
         for history in self:
             history.avatar_128 = history.partner_id.avatar_128 or history.guest_id.avatar_128
+
+    # ===================================================================
+    # REPORTING
+    # ===================================================================
+
+    @api.depends("call_history_ids")
+    def _compute_has_call(self):
+        for history in self:
+            history.has_call = 1 if history.call_history_ids else 0
+
+    @api.depends("call_history_ids.duration_hour")
+    def _compute_call_duration_hour(self):
+        for history in self:
+            history.call_duration_hour = sum(history.call_history_ids.mapped("duration_hour"))
+
+    @api.depends(
+        "channel_id.livechat_agent_requesting_help_history",
+        "channel_id.livechat_agent_providing_help_history",
+    )
```

Comment:
not sure if important but dependencies currently don't follow all relations like in the compute itself